processing/data_collection/gazette/spiders/ba_camacari.py

```diff
@@ -0,0 +1,53 @@
+import re
+import datetime
+from urllib.parse import urlencode
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaCamacari(BaseGazetteSpider):
+    TERRITORY_ID = "2905701"
+    BASE_URL = "http://www.camacari.ba.gov.br/wp-content/themes/camacari/assets/ajax/arquivos.php"
+    name = "ba_camacari"
+    allowed_domains = ["camacari.ba.gov.br"]
+    start_date = datetime.date(2007, 10, 10)
+
+    def start_requests(self):
+        params = {"paged": 1, "categoria": "diario-oficial"}
+        url = f"{self.BASE_URL}?{urlencode(params)}"
+        yield scrapy.Request(url=url, callback=self.parse, meta={"params": params})
+
+    def parse(self, response):
+        entries = response.css(".col-sm-12")
+
+        for entry in entries:
+            date_text = entry.css(".mb-10::text").re_first(r"\d+\/\d+\/\d+")
+            date = dateparser.parse(date_text, languages=["pt"]).date()
+
+            file_url = entry.css(".mb-40 a::attr(href)").get()
+
+            title = entry.css(".mb-40 a::text").get()
+            edition_number = re.search(r"\d+", title.replace(".", "")).group(0)
+            is_extra_edition = bool(re.search(r"EXTRA", title, re.IGNORECASE))
```

Comment:
I would like to move the string with css selector and regex to variables with names describing what elements they are filtering.

Reply:
Done!